### PR TITLE
Feat/get live data for bolnisi

### PR DESCRIPTION
**Pull request by Kammerlo**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -461,6 +467,30 @@
...
+                    <execution>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <inputSpec>${project.basedir}/src/main/resources/openapi/cf-product-tracing.json
+                            </inputSpec>
+                            <generatorName>spring</generatorName>
+                            <modelPackage>com.example.api.model</modelPackage>
+                            <apiPackage>com.example.api.client</apiPackage>
```
Comment (nemo83) [inline pom.xml:484]: this should be replaced with cardanofoundation path please

File: src/main/java/org/cardanofoundation/explorer/api/service/impl/BolnisiMetadataServiceImpl.java [hunk 2]
```diff
@@ -199,13 +207,10 @@ public CertDetailsData getCertDetailsData(String txHash, String certNo) {
...
   @Override
   public BolnisiProjectNumberResponse getBolnisiProjectNumber() {
-    BolnisiProjectNumberResponse response = new BolnisiProjectNumberResponse();
-    // Hardcoded values for the number of bottles, wineries, and certificates
-    // TODO
-    response.setNumberOfBottles(BigInteger.valueOf(71683));
-    response.setNumberOfWineries(BigInteger.valueOf(29));
-    response.setNumberOfCertificates(BigInteger.valueOf(1));
-    return response;
+    RestTemplate template = new RestTemplate();
```
Comment (nemo83) [inline src/main/java/org/cardanofoundation/explorer/api/service/impl/BolnisiMetadataServiceImpl.java:210]: we have both `restTemplate` and `webClient` any reasons why we need both?

**Review by nemo83 — CHANGES_REQUESTED** [review]
Great work here, open api for the win!

Please update the packages name!

The restTemplate/webClient is a minor one

File: src/main/java/org/cardanofoundation/explorer/api/service/impl/BolnisiMetadataServiceImpl.java (hunk 2 above)
Comment (Kammerlo) [reply src/main/java/org/cardanofoundation/explorer/api/service/impl/BolnisiMetadataServiceImpl.java:210]: Good point, will replace it with webclient.

Comment (Kammerlo) [reply src/main/java/org/cardanofoundation/explorer/api/service/impl/BolnisiMetadataServiceImpl.java]: Done

**Review by nemo83 — APPROVED** [review]
LGTM send it 🚀

**Pull request merged**